lib/zenv.ts

```diff
@@ -4,10 +4,14 @@ const ZEnv = z.object({
   NODE_ENV: z.string().min(1),
   NEXT_PUBLIC_CLIENT_ID: z.string().min(1),
   NEXT_PUBLIC_API_KEY: z.string().min(1),
+  NEXT_PUBLIC_ATTACHMENT_MAX_SIZE_MB: z.number(),
+  NEXT_PUBLIC_ATTACHMENT_MAX_FILES: z.number().min(1),
```

Comment:
Can we document what these numbers are from Google?  If they are imposed by
Google, let's just hard code them instead of making them env variables.

Reply:
As per this link: https://support.google.com/mail/answer/6584#zippy=%2Cattachment-size-limit, there is a max 25MB in total file size across all attachments.

I have never known a limit on the number of attachments.  Can we program it with that logic instead and document via the above link?

Reply:
> As per this link: https://support.google.com/mail/answer/6584#zippy=%2Cattachment-size-limit, there is a max 25MB in total file size across all attachments.
> 
> I have never known a limit on the number of attachments. Can we program it with that logic instead and document via the above link?

Okay. I'll use a hard coded max total size.

Reply:
Please check this loom (Set max 2MB size to test easier)

https://www.loom.com/share/f1c0be333d5f435fb2e8406c9317aee9

Reply:
Thanks -- but please don't put the error on the toast.  Instead, show it on the upload section and turn it red.

Reply:
> Thanks -- but please don't put the error on the toast. Instead, show it on the upload section and turn it red.

I came up doing this.
https://www.loom.com/share/9d3307413bd343258c5b4089ab14cc4d